scylla-cdc/src/consumer.rs

```diff
@@ -467,4 +494,57 @@ mod tests {
         assert_eq!(schema.deleted_mapping.len(), 1);
         assert_eq!(schema.deleted_el_mapping.len(), 0);
     }
+
+    #[tokio::test]
+    async fn test_take_value() {
+        let session = setup().await.unwrap();
+        let result = session
+            .query(
+                format!("SELECT * FROM {};", TEST_SINGLE_VALUE_CDC_TABLE),
+                (),
+            )
+            .await
+            .unwrap();
+
+        let row = result.rows.unwrap().remove(0);
+        let schema = CDCRowSchema::new(&result.col_specs);
+        let mut cdc_row = CDCRow::from_row(row, &schema);
+
+        assert_eq!(cdc_row.take_value("v").unwrap().as_int().unwrap(), 3);
+        assert!(cdc_row.take_value("no_such_column").is_none());
+    }
+
+    #[tokio::test]
+    async fn test_take_deleted_elements() {
+        let session = setup().await.unwrap();
+        session
+            .query(
+                format!(
+                    "UPDATE {} SET vs = vs - {{{}}} WHERE pk = {} AND ck = {}",
+                    TEST_SINGLE_COLLECTION_TABLE, 2, 1, 2
```

Comment:
Don't interpolate query parameters using `format`. Instead, use bind markers (`?`) and provide values in the designated parameter of the `query` method.

It should be possible to convert every parameter here with the exception of the table name.